file-upload-tutorial.md

```diff
@@ -0,0 +1,25 @@
+# A tutorial on doing file uploads in Hapi
+
+### Why file uploads
+
+Every website needs content, but not every website needs to create its own content. If you want your users to be able to share images, music or video with their friends and with the world, you have to crack file uploads.
+
+### What we made
+
+We have built a simple website using the Hapi framework for Node.js. It provides a form for the users that allows them to submit files to the server. The server then saves that in a 'pix' directory (though the files do not have to be pictures!).
+
+To aid the rest of our FAC cohort, we have developed a repo that provides a tutorial on building the site.
+
+### On Modules
+
+Certain modules can help in the parsing of complicated input to the server -- in this case, the multipart form is the complex data. Some examples make use of the 'multiparty' node module to help with parsing it. We have chosen to abstain from doing so. There are three reasons why:
```

Comment:
Suggest adding a clarification of 'multipart form' or link to an explanation (usually a good answer to a stackoverflow question or article online) - seeing as this is for a wide audience that includes beginners